taskstats/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "taskstats"
+version = "0.1.0"
+edition = "2021"
+authors = ["Michael Hupfer <michael.hupfer@aox.de>"]
+license = "MIT"
+description = "Library for handling of Linux task statitisc information"
+keywords = ["taskstats"]
+categories = ["netlink", "Linux"]
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+netlink-sys = { version = "0.8.3", path = "../netlink-sys", features = ["tokio_socket"] }
+netlink-packet-core = { version = "0.4.2", path = "../netlink-packet-core" }
+netlink-packet-generic = { version = "0.3.1", path = "../netlink-packet-generic" }
+netlink-packet-utils = { version = "0.5.1", path = "../netlink-packet-utils" }
+serde = { version = "1.0", features = ["derive"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
Please replace this `full` into real features you need to save the compiling time of everyone.